docs/samples/Microsoft.ML.AutoML.Samples/Microsoft.ML.AutoML.Samples.csproj

```diff
@@ -16,7 +16,7 @@
     </ProjectReference>
 
     <NativeAssemblyReference Include="MatrixFactorizationNative" />
-    <NativeAssemblyReference Include="FastTreeNative" />
+    <NativeAssemblyReference Condition="'$(TargetArchitecture)' != 'arm64' And '$(TargetArchitecture)' != 'arm'" Include="FastTreeNative" />
```

Comment:
I wonder if we can create a helper property in Directory.Build.props `$(TargetsArm)` or similar. Then we don't need this duplicated everywhere.

Reply:
Yep, good idea. I'll go ahead and do that.

Reply:
Ok I have added this. Take a look and see if it was what you were thinking. @safern can you take a look as well since you know msbuild really well? `Directory.Build.props` lines 30 - 37.